### DISCUSSION TITLE
Welcome to tinycore-redpill Discussions!

### DISCUSSION THREAD

**Discussion by PeterSuh-Q3** [PeterSuh-Q3/tinycore-redpill#2] [Q&A] [answered]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by rikikhaastrecht** [comment]
hi Peter,
need some help with NVME support with 7.2 for DVA3221?
whats the ssh command to add my drives?

P: /devices/pci0000:00/0000:00:1b.0/0000:04:00.0/nvme/nvme0/nvme0n1
N: nvme0n1
E: DEVNAME=/dev/nvme0n1
E: DEVPATH=/devices/pci0000:00/0000:00:1b.0/0000:04:00.0/nvme/nvme0/nvme0n1
E: DEVTYPE=disk
E: MAJOR=259
E: MINOR=1
E: PHYSDEVBUS=pci
E: PHYSDEVDRIVER=nvme
E: PHYSDEVPATH=/devices/pci0000:00/0000:00:1b.0/0000:04:00.0
E: SUBSYSTEM=block
E: SYNO_ATTR_SERIAL=S4EVNJ0N206902W
E: SYNO_DEV_DISKPORTTYPE=CACHE
E: SYNO_INFO_PLATFORM_NAME=denverton
E: SYNO_KERNEL_VERSION=4.4
E: SYNO_SUPPORT_USB_PRINTER=yes
E: SYNO_SUPPORT_XA=no
E: TAGS=:systemd:
E: USEC_INITIALIZED=691943

P: /devices/pci0000:00/0000:00:1b.0/0000:04:00.0/nvme/nvme0/nvme0n1
N: nvme0n1
E: DEVNAME=/dev/nvme0n1
E: DEVPATH=/devices/pci0000:00/0000:00:1b.0/0000:04:00.0/nvme/nvme0/nvme0n1
E: DEVTYPE=disk
E: MAJOR=259
E: MINOR=1
E: PHYSDEVBUS=pci
E: PHYSDEVDRIVER=nvme
E: PHYSDEVPATH=/devices/pci0000:00/0000:00:1b.0/0000:04:00.0
E: SUBSYSTEM=block
E: SYNO_ATTR_SERIAL=S4EVNJ0N206902W
E: SYNO_DEV_DISKPORTTYPE=CACHE
E: SYNO_INFO_PLATFORM_NAME=denverton
E: SYNO_KERNEL_VERSION=4.4
E: SYNO_SUPPORT_USB_PRINTER=yes
E: SYNO_SUPPORT_XA=no
E: TAGS=:systemd:
E: USEC_INITIALIZED=691943

**Reply by PeterSuh-Q3** [reply]
Is "support_m2_pool=yes" checked in /etc.defaults/synoinfo.conf file?
If this value is no or there is no key, you are not using the latest version of M SHELL.

**Reply by PeterSuh-Q3** [reply]
An error in which the NVMe cache is not activated in the DVA3221 was confirmed, so additional supplements were made.
Please check the link below for details.

https://xpenology.com/forum/topic/13342-nvme-cache-support/?do=findComment&comment=446408

**Comment by digilabs84** [comment]
Hello Peter , 

I install success 920+ 7.2 , but can not sign in Synology ID , i use the serial in boot screen to signin website Synology but it's showed wrong . How can i sign in ID for using ? thank you !

**Reply by PeterSuh-Q3** [reply]
I'm having a hard time understanding any of your questions.
There are only incomplete, unknown words.
I'm guessing you were trying to connect to TTYD using port 7681 via a web browser?

**Comment by hbixu** [comment]
Hello Peter, 

What is the recommended procedure for updating DSM?
Is there any difference in the update procedure between version updates (7.1.0 to 7.2.0) or minor correction updates (same version, but update 1, update 2, ...)?

Thanks for your work.

**Reply by PeterSuh-Q3** [reply, marked as answer]
Simply build the loader for the version you want. When you prepare DSM 7.2.1, the loader will automatically patch up to the latest update 4 [current final smallfixversion]. This is done only once, at the time you build the loader. Update 5 will not be automatically installed on subsequent boots.

**Reply by hbixu** [reply]
Thank you for your answer. 
I applied Update 4 [current final version of minor fix] and it was applied without any problem.
So, in the future, when an Update 5 is released, do I have to build a new loader in order to get an updated loader? This before applying Update 5?
